src/utils/tableReporter.ts

```diff
@@ -0,0 +1,64 @@
+import Mocha from "mocha";
+
+import { decorators } from "../utils/colors";
+import { CreateLogTable } from "../utils/tableCli";
+
+const { EVENT_RUN_END, EVENT_TEST_FAIL, EVENT_TEST_PASS } =
+  Mocha.Runner.constants;
+
+interface TableReporterProps {
+  runner: Mocha.Runner;
+  stats: Mocha.Stats;
+  on: any;
+}
+
+class TableReporter {
+  constructor(runner: TableReporterProps) {
+    const stats = runner.stats!;
+
```

Comment:
Since the LogTable appears when all tests are finished, maybe you should add an "announcement" that currently tests are running - so that the user is aware that something is happening at the background.